fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseVisitor.java

```diff
@@ -250,6 +250,20 @@ public Void visitModifyTablePropertiesClause(ModifyTablePropertiesClause clause,
                         "Property " + PropertyAnalyzer.PROPERTIES_REPLICATED_STORAGE +
                                 " must be bool type(false/true)");
             }
+            if (table instanceof OlapTable) {
+                OlapTable olapTable = (OlapTable) table;
+                boolean hasGIN = false;
+                for (Index index : olapTable.getIndexes()) {
+                    if (index.getIndexType() == IndexDef.IndexType.GIN) {
+                        hasGIN = true;
+                        break;
+                    }
+                }
+                if (properties.get(PropertyAnalyzer.PROPERTIES_REPLICATED_STORAGE).equalsIgnoreCase("true") && hasGIN) {
+                    ErrorReport.reportSemanticException(ErrorCode.ERR_COMMON_ERROR,
```

Comment:
Put the error message into ErrorCode.java

Reply:
Fixed